README.md

```diff
@@ -12,8 +12,11 @@ stable version.
 
 ## Getting started
 
+1. Install [depot_tools](https://commondatastorage.googleapis.com/chrome-infra-docs/flat/depot_tools/docs/html/depot_tools_tutorial.html#_setting_up)
 1. Install [CMake](https://cmake.org/install/)
 1. Install [python3](https://www.python.org/downloads/)
+1. Copy `<src path>/tools/standalone.gclient` to `<src path>/.gclient`
```

Comment:
Why don't we check in a .gclient file instead of this?

Reply:
From my understanding, it makes things easier if this gets embedded into another project. There is no .gclient file in the general case, if you want to use this standalone you copy over the file.

Reply:
Hm ok, I would think only the root .gclient would do anything so it'd just be noise if in the subdir but it's less steps for development on subspace.

Reply:
done